erpnext/stock/doctype/inventory_dimension/test_inventory_dimension.py

```
@@ -234,6 +234,7 @@ def test_check_mandatory_dimensions(self):
 			)
 		)
 
+		doc.load_from_db
```

Ruff (0.15.11)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)